analytics.js

```diff
@@ -833,11 +854,16 @@ class Analytics {
       checkReservedKeywords(rudderElement.message, type);
 
       // if not specified at event level, All: true is default
-      const clientSuppliedIntegrations = rudderElement.message.integrations || { 'All' : true };
+      const clientSuppliedIntegrations = rudderElement.message.integrations || {
+        All: true,
+      };
 
       // structure user supplied integrations object to rudder format
       tranformToRudderNames(clientSuppliedIntegrations);
-      rudderElement.message.integrations = clientSuppliedIntegrations;
+      rudderElement.message.integrations = {
+        ...this.integrationsData,
+        ...clientSuppliedIntegrations,
+      };
```

Comment:
This should be done right before sending it to the cloud mode. Move this after Ln 921. Also, naming transformation should happen for this to work (this is happening in Ln 921).

Reply:
updayed